docs/pages/2019_IIT_Mandi_India/sections/setup.rst

```diff
@@ -34,7 +34,7 @@ It's recommended for you to place the ssh key you received in a folder dedicated
    ``chmod 600 ~/.ssh/aiida_tutorial_NUM``. You can check check with ``ls -l``
    that the permissions of this file are now ``-rw-------``.
 
-After that ssh key is in place, you can add the following block your ``~/.ssh/config`` file:
+After the ssh key file is in place, add the following block to your ``~/.ssh/config`` file:
```

Comment:
```suggestion
After the ssh key files are in place, add the following block to your ``~/.ssh/config`` file:
```
This is related to the comments from PR #171

Reply:
So participants **do** receive both the private and public key file?

Reply:
Yup - you may check that e-mail now .. 
![image](https://66.media.tumblr.com/71a4bd377324368fc7f98cc3d6664748/tumblr_inline_nfrcazWZHJ1rkygkg.gif)

Reply:
Will fix.